qutip/solver/ode/qutip_integrator.py

```diff
@@ -174,14 +174,20 @@ def get_state(self, copy=True):
         return self._t, _data.matmul(self.U, _data.dense.Dense(self._y))
 
     def set_state(self, t, state0):
+        self._is_set = True
         self._t = t
         self._y = _data.matmul(self.Uinv, state0).to_array()
         self._is_set = True
```

Comment:
I am not sure what `self._is_set` is meant to do but I find it weird that it is "set" twice. Can you check that both are needed?

Reply:
This was a mistake, fixing.